app/views/coronavirus_landing_page/components/landing_page/_statistics_section.html.erb

```diff
@@ -1,80 +1,151 @@
-      text: topic_section["header"],
-      margin_bottom: 6,
-      font_size: "m"
-    } %>
-  </div>
+<%= render "govuk_publishing_components/components/heading", {
+  text: topic_section["header"],
+  margin_bottom: 6,
+  font_size: "m"
+} %>
 
-  <div class="govuk-grid-column-two-thirds">
-    <% link = topic_section["links"].shift %>
-    <%= link_to(
-      link["label"].html_safe,
-      link["url"],
-      class: 'covid__topic-list-link govuk-link',
-      data: {
-        module: "gem-track-click",
-        track_category: "pageElementInteraction",
-        track_action: "Statistics",
-        track_label: link["url"],
-      },
-    ) %>
+<% link = topic_section["links"].shift %>
+<p class="govuk-body govuk-!-margin-bottom-6">
+  <%= link_to(
+    link["label"].html_safe,
+    link["url"],
+    class: 'govuk-link',
+    data: {
+      module: "gem-track-click",
+      track_category: "pageElementInteraction",
+      track_action: "Statistics",
+      track_label: link["url"],
+    },
+  ) %>
+</p>
 
-    <% if statistics %>
-      <% if statistics.new_positive_tests %>
-        <p><%= topic_section.dig("tests", "title") %></p>
-        <p><%= topic_section.dig("tests", "daily_heading") %></p>
-        <p><%= number_with_delimiter(statistics.new_positive_tests) %></p>
-        <p><%= topic_section.dig("tests", "weekly_heading") %></p>
-        <p><%= number_with_delimiter(statistics.current_week_positive_tests) %></p>
-        <p><%= number_with_delimiter(statistics.current_week_positive_tests_change_number) %></p>
-        <p><%= number_to_percentage(statistics.current_week_positive_tests_change_percentage, precision: 1) %>
-        <p><%=topic_section.dig("tests", "guidance_link") %></p>
-        <p><%=topic_section.dig("updated_date") %> <%= statistics.new_positive_tests_date %></p>
-      <% end %>
+<% if statistics %>
+  <div class="govuk-grid-row">
+    <% if statistics.new_positive_tests %>
+      <div class="govuk-grid-column-one-third">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: topic_section.dig("tests", "title"),
+          margin_bottom: 4,
+          font_size: "s",
+          heading_level: 3
+        } %>
+        <p class="govuk-body">
+          <%= topic_section.dig("tests", "daily_heading") %><br/>
+          <span class="covid-statistics__emphasis"><%= number_with_delimiter(statistics.new_positive_tests) %></span>
+        </p>
+        <p class="govuk-body">
+          <%= topic_section.dig("tests", "weekly_heading") %><br/>
+          <span class="covid-statistics__emphasis"><%= number_with_delimiter(statistics.current_week_positive_tests) %></span>
+          <span class="govuk-tag govuk-tag--yellow covid-statistics__tag">
+            <% tests_number = number_with_delimiter(statistics.current_week_positive_tests_change_number).to_i %>
+            <%= render partial: 'coronavirus_landing_page/components/landing_page/arrow', locals: { number: tests_number } %>
```

Comment:
`statistics.current_week_positive_tests_change_number` is already a number, so `to_i` doesn't need to be applied to it. The value as is can be passed directly to the "arrow" partial:
```suggestion
            <%= render partial: 'coronavirus_landing_page/components/landing_page/arrow', locals: { number: statistics.current_week_positive_tests_change_number } %>
```

Reply:
The same applies for "hospital admissions".

Reply:
Thanks, have updated. It's a bit less contorted now.

Interestingly this highlighted a bug - I was using `number_with_delimiter` to convert the number from e.g. `1234` to `1,234` and then applying rails `.to_i` method to convert it back into a number. Unfortunately that doesn't work properly - instead of converting `1,234` to `1234` it converts it to `1` (or `1.0` if using `.to_f`). Bizarre.